inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -1723,6 +1734,15 @@ void MKLDNNEltwiseNode::fillScalesAndShifts() {
     }
 }
 
+void MKLDNNEltwiseNode::fuseInto(MKLDNNNodePtr& parentNode) {
+    bool isSpecialConvolutionAddFusing = parentNode->getType() == Convolution && getAlgorithm() == EltwiseAdd &&
+            getParentEdgesAtPort(0)[0]->getDims().ToSizeVector() == getParentEdgesAtPort(1)[0]->getDims().ToSizeVector();
```

Comment:
is it case conv + bias?
shouldn't we check channels equality or check that second input is scalar value?

Reply:
This is special Convolution + Add (full shape) case which is handled via append_sum() routine. I left the comment.